.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -57,7 +57,7 @@ body:
     id: link-to-sentry
     attributes:
       label: Link to Sentry event
-      description: If applicable, provide a link to the affected event from your Sentry account. The event will only be viewable by Sentry staff. **Note**: This should be an _event_ link, not an issue link, and should match the form shown below. It can be found by clicking on the event ID on the issue details page.
+      description: If applicable, provide a link to the affected event from your Sentry account. The event will only be viewable by Sentry staff.
```

Comment:
Why remove this? (It's something even our staff get wrong without periodic reminders.)